packages/cli/src/cli.ts

```diff
@@ -38,7 +38,7 @@ export const main = async () => {
         alias: "help",
         type: "boolean",
       })
-      .scriptName("webstudio")
+      .scriptName("webstudio-cli")
```

Comment:
@kof @istarkov i changed this to `webstudio-cli` since the bin is changed to `webstudio-cli` too in package.json

Or it's going to trigger for both ?